File: WOT-iOS/WOTData/WOTData/CoreData/model/autogenerated/VehicleprofileArmorList+CoreDataProperties.m
//
//  VehicleprofileArmorList+CoreDataProperties.m
//  WOTData
//
//  Created by Pavel Yeshchyk on 1/17/20.
//  Copyright © 2020 Pavel Yeshchyk. All rights reserved.
//
//

#import "VehicleprofileArmorList+CoreDataProperties.h"

@implementation VehicleprofileArmorList (CoreDataProperties)

+ (NSFetchRequest<VehicleprofileArmorList *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:@"VehicleprofileArmorList"];
}

@dynamic hull;
@dynamic turret;
@dynamic vehicleprofile;

@end

File: WOT-iOS/WOTData/WOTData/CoreData/model/autogenerated/VehicleprofileArmorList+CoreDataProperties.swift
//
//  VehicleprofileArmorList+CoreDataProperties.swift
//
//
//  Created by Pavel Yeshchyk on 4/23/20.
//
//

import Foundation
import CoreData

extension VehicleprofileArmorList {
    @nonobjc public class func fetchRequest() -> NSFetchRequest<VehicleprofileArmorList> {
        return NSFetchRequest<VehicleprofileArmorList>(entityName: "VehicleprofileArmorList")
    }

    @NSManaged public var hull: VehicleprofileArmor?
    @NSManaged public var turret: VehicleprofileArmor?
    @NSManaged public var vehicleprofile: Vehicleprofile?
}
